pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -64,6 +64,18 @@ func (b Bottlerocket) Script() (string, error) {
 		if b.KubeletConfig.EvictionHard != nil {
 			s.Settings.Kubernetes.EvictionHard = b.KubeletConfig.EvictionHard
 		}
+		if b.KubeletConfig.EvictionMaxPodGracePeriod != nil {
```

Comment:
Can we add a test (at least a functional one) for the changes that you are making here. There may also be a change that you could make to the current E2E integration testing in our Kubelet Config suite tests that would extend eviction soft testing for BR

Reply:
I'll try to take a look at this in the next few days